tests/integration/features/fixtures/cluster_test_fixtures.go

```diff
@@ -105,12 +109,19 @@ func GetFeatureTracker(ctx context.Context, cli client.Client, appNamespace, fea
 	return tracker, err
 }
 
-func NewDSCInitialization(ns string) *dsciv1.DSCInitialization {
-	return &dsciv1.DSCInitialization{
+func NewDSCInitialization(ctx context.Context, cli client.Client, ns string) *dsciv1.DSCInitialization {
+	dsci := &dsciv1.DSCInitialization{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: gvk.DSCInitialization.Version,
+			Kind:       gvk.DSCInitialization.Kind,
+		},
 		ObjectMeta: metav1.ObjectMeta{
 			Name: "default-dsci",
 		},
-		Spec: dsciv1.DSCInitializationSpec{
+	}
+
+	_, errCreate := controllerutil.CreateOrUpdate(ctx, cli, dsci, func() error {
```

Comment:
Why it's needed?

Reply:
To have proper UID created by `kube-apiserver`, as DSCI is now used as owner-ref.

Reply:
Got it. I'm not big fan of mixing allocating and k8s creation in New* function (even if it causes less code changes), but it's up to your conventions :)

Reply:
Fair enough, but that's a different question :) We can split it up.